inc/server/class-form-server.php

```diff
@@ -176,6 +178,26 @@ public function register_routes() {
 				),
 			)
 		);
+		register_rest_route(
+			$namespace,
+			'/form/confirm',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'confirm_form' ),
+					'permission_callback' => function ( $request ) {
+						return __return_true();
+					},
+					'args'                => array(
+						'record_id' => array(
+							'validate_callback' => function( $param, $request, $key ) {
+								return is_numeric( $param );
+							},
+						),
+					),
+				),
+			)
+		);
```

Comment:
If this is to confirm the submission, why is it a GET request with an open permission callback?

Reply:
For fast testing. I was thinking of adding a license check; what do you think?

Reply:
@Soare-Robert-Daniel I'm not sure it's a good idea to send the license from frontend for validation, we expose the license that way to visitors. Maybe we can create a nonce for that particular form submission and send it with Stripe request, along side checkout ID, and validate it with that?

Reply:
By license verification, I meant at the server level. To be a PRO-only feature. I will add a dynamic nonce generation.